matriculation-plugin/src/main/resources/META-INF/resources/scripts/gui/matriculation/matriculation.jsx

```diff
@@ -976,8 +1040,12 @@ class App extends React.Component {
                   amountOfMandatoryAttendances={ this.getAmountOfMandatoryAttendances() }
                   amountOfAcademicSubjectAttendances={ this.getAmountOfAcademicSubjectAttendances() }
                   amountOfMandatoryAdvancedSubjectAttendances = { this.getAmountOfMandatoryAdvancedSubjectAttendances() }
+                  isConflictingMandatory={(attendance) => { return this.isConflictingMandatory(attendance);}}
                   conflictingAttendances={this.isConflictingAttendances()}
+                  isConflictingRepeat={(attendance) => { return this.isConflictingRepeat(attendance);}}
```

Comment:
you should pass `this.isConflictingRepeat` alone as it is basically equivalent

If you are having trouble with context being undefined you need to do

`this.isConflictingRepeat = this.isConflictingRepeat.bind(this)` at constructor time, this is an issue from react that comes from javascript changing contexts.

Reply:
The same applies to the other attributes written in a similar fashion.